.github/workflows/deploy-daac.yml

```diff
@@ -56,6 +57,7 @@ jobs:
               job_spec/INSAR_ISCE_MULTI_BURST.yml
               job_spec/OPERA_DISP_TMS.yml
               job_spec/ARIA_S1_GUNW.yml
+              job_spec/OPERA_DISP_TMS.yml
```

Comment:
```suggestion
```
Duplicate of Line 58, no?

Reply:
Thanks, I recalled we added a non-functional copy a month ago but thought we'd rolled it back out.